### DISCUSSION TITLE
How could I return name of select, when im clearing last selected item in multi-select?

### DISCUSSION THREAD

**Discussion by fed8055** [JedWatson/react-select#4856] [Q&A] [answered]
I recieving an empty array when im clearing multi-select, but I need to get element name. How can I do it? 
![image](https://user-images.githubusercontent.com/65348350/136761679-775ad312-cf55-4176-a439-4c567378542b.png)

**Comment by Rall3n** [comment, marked as answer]
Hello @fed8055,

the `name` prop is part of the second argument received by the `onChange` prop.
You can easily access it as follows:

```jsx
<Select
  name="test"
  onChange={(value, meta) => {
    let name = meta.name;
  }}
/>
```
